core/src/test/java/org/infinispan/security/DynamicRBACTest.java

```diff
@@ -0,0 +1,114 @@
+package org.infinispan.security;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.security.PrivilegedAction;
+import java.security.PrivilegedExceptionAction;
+import java.util.Map;
+
+import javax.security.auth.Subject;
+
+import org.infinispan.Cache;
+import org.infinispan.configuration.cache.AuthorizationConfigurationBuilder;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalAuthorizationConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfigurationBuilder;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.security.mappers.ClusterPermissionMapper;
+import org.infinispan.security.mappers.ClusterRoleMapper;
+import org.infinispan.test.SingleCacheManagerTest;
+import org.infinispan.test.TestingUtil;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.Test;
+
+@Test(groups="functional", testName="security.DynamicRBACTest")
+public class DynamicRBACTest extends SingleCacheManagerTest {
```

Comment:
probably worth converting to multiple cache manager to check everything is serializable/stored properly :)

Reply:
Done